mlflow/paddle/__init__.py

```diff
@@ -234,6 +240,11 @@ def load_model(model_uri):
             - ``models:/<model_name>/<model_version>``
             - ``models:/<model_name>/<stage>``
 
+    :param model: Required when loading a `paddle.Model` model.
+                  Must be an instance of `paddle.Model` saved with `training=True`.
```

Comment:
```suggestion
    :param model: Required when loading a `paddle.Model` model.
                  Must be an instance of `paddle.Model` saved with `training=True`.
```

The current error message sounds like we need to call `model.save` before `model.load`, but that's not true right?

Reply:
Yes, then what about 
```
Required when loading a `paddle.Model` model.
                  Must be an instance of `paddle.Model` saved including files .pdopt and .pdparams. This can be achieved by set `training=True` when saved.
```

Reply:
What about fixing the first sentence?

> Required when loading a `paddle.Model` model saved with `training=True`.

Reply:
About checking if the model type supports re-train.
If the model is saved with `training=True`, the model will be saved in the two files, .pdopt and .pdparams. 
If set `training=False`, the model will be saved in three files, .pdiparams, .pdiparams.info and .pdmodel
When we are checking if the model supports re-train, can we just check if .pdparams exists? If .pdparams has issues or .pdopt doesn't exist, the paddle api will return error message.

Reply:
> What about fixing the first sentence?
> 
> > Required when loading a `paddle.Model` model saved with `training=True`.

Year, cool